pkg/loader/load.go

```diff
@@ -624,6 +629,82 @@ func (b *batchManager) put(txn *Txn) error {
 	return nil
 }
 
+type txnManager struct {
```

Comment:
any test for this object?

Reply:
If in the end we decided to use `txnManager`, I will add some tests.